website/content/docs/integrations/consul/service-mesh.mdx

```diff
@@ -134,6 +134,43 @@ service_prefix "" { policy = "read" }
 node_prefix    "" { policy = "read" }
 ```
 
+#### Transparent Proxy
+
+Using Nomad's support for [transparent proxy][] configures the network namespace
```

Comment:
how about adding the _groups_ network namespace to clarify all tasks in the group will be subject to the tproxy